_pages/decide/comparative-analysis.md

```diff
@@ -10,23 +10,23 @@ A detailed review of existing experiences provided either by direct competitors
 
 ## Reasons to use it
 
-To identify competitors’ solutions that excel, are lacking, or are missing critical design elements. Comparative analysis can give new solutions a competitive edge by identifying areas of opportunity, gaps in experience offerings, and potential design patterns to adopt or avoid.
+To identify competitors’ solutions that excel, are lacking, or are missing critical design elements. Comparative analysis can give you a competitive edge by identifying opportunities, gaps in other services, and potential design patterns to adopt or avoid.
 
 ## Time required
 
-**Medium:** 1–2 hours per identified competitor. One hour to analyze each competitor. 4–8 hours total to generate a report.
+**Medium:** 1–2 hours to analyze and write a report about each competitor.
```

Comment:
Could change to "evaluation" if that seems smaller/more appropriate? I haven't seen a completed comparative analysis, so I'm not sure how big they are.